### PR TITLE
Jboss Fuse Camel-k zstream 4.16 and firewatch updates

### PR DESCRIPTION
**Pull request by smatula**

### CODE REVIEW

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-jboss-fuse-camel-k-test-container-main-camel-k-ocp4.15-lp-interop-camel-k-interop-ibmcloud periodic-ci-jboss-fuse-camel-k-test-container-main-camel-k-ocp4.16-lp-interop-camel-k-interop-ibmcloud

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-jboss-fuse-camel-k-test-container-main-camel-k-ocp4.15-lp-interop-camel-k-interop-ibmcloud

File: ci-operator/config/jboss-fuse/camel-k-test-container/jboss-fuse-camel-k-test-container-main__camel-k-ocp4.16-lp-interop.yaml [hunk 1]
```diff
@@ -61,6 +59,35 @@ tests:
...
+    env:
+      BASE_DOMAIN: zstream.interop.ccitredhat.com
+      FIREWATCH_CONFIG: |
+        {
+          "failure_rules":
+            [
+              {"step": "camel-k-interop-tests", "failure_type": "test_failure", "classification": "Test Failure", "jira_project": "CMLK", "group": {"name": "lp-tests", "priority": 2}, "jira_additional_labels": ["!default","interop-tests"]},
+              {"step": "camel-k-interop-tests", "failure_type": "pod_failure", "classification": "Test Execution", "group": {"name": "lp-tests", "priority": 2}, "jira_additional_labels": ["!default","interop-tests"]}
+            ],
+          "success_rules":
```
Comment (oharan2) [inline ci-operator/config/jboss-fuse/camel-k-test-container/jboss-fuse-camel-k-test-container-main__camel-k-ocp4.16-lp-interop.yaml:75]: You have `success_rules` in https://raw.githubusercontent.com/CSPI-QE/cspi-utils/main/firewatch-base-configs/ibm-ipi/lp-interop.json :)

**Review by oharan2 — APPROVED** [review]

**Pull request merged**